openapi/controllers/main.py

```diff
@@ -39,3 +49,77 @@ def copy(obj, **d):
         }})
 
         return result
+
+
+class OAS(http.Controller):
+
+    @http.route('/api/v1/<namespace_name>/swagger.json',
```

Comment:
Why not use /api/v1/<namespace_name>.json

Reply:
I've seen `swagger.json` or `swagger_version_description.json`, yet most of the time `swagger` is (still) common practice.

Reply:
Akin to the other comments:
Most of the time a swagger file is consumed as per it's URL, so: 
`/api/v1/<namespace_name>/swagger.json'` has it all.